src/server/req_delete.c

```diff
@@ -80,6 +80,7 @@
 #include "pbs_nodes.h"
 #include "svrfunc.h"
 
+#define QDEL_DURATION 5
```

Comment:
Add a comment here too or have a better name that explains the need for this variable. Something on the lines of QDEL_BREAKER_SECS would be better.